src/pages/ItemDetail/ItemDetail.js

```diff
@@ -1,8 +1,171 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './ItemDetail.scss';
 
 const ItemDetail = () => {
-  return <div />;
+  const [itemData, setItemData] = useState({});
+  // 아이템 데이터 state구조
+  const [currentTab, setCurrentTab] = useState('상품상세');
+  // 현재 탭 state 구조
+  const [quantity, setQuantity] = useState(1);
+  // 수량 데이터 state구조
+  useEffect(() => {
+    fetch('data/itemData.json')
+      .then(res => res.json())
+      .then(data => setItemData(data));
+  }, []);
+  //목 데이터를 가져와서 아이템 데이터로 설정
+  const {
+    first_category,
+    second_category,
+    title,
+    description,
+    discount,
+    price,
+    thumbnail_images,
+    detail_images,
+  } = itemData;
+  //구조 분해 할당하여 가독성 향상
+
+  const plusPrice = () => {
+    setQuantity(quantity + 1);
+  };
+  const minusPrice = () => {
+    setQuantity(quantity - 1);
+  };
+  //수량 1씩 증가 및 1씩 감소
+  const disabledMinus = quantity < 2;
+  //수량이 2보다 작을때 마이너스 안눌리게 설정
+  const updateCurrentTab = ({ target }) => {
+    const currentTabName = target.innerHTML;
+    setCurrentTab(currentTabName);
+  };
+  //현재 탭에 이름을 부여하도록 설정
+  const discountPrice = price * ((100 - discount) / 100);
+
+  return Object.keys(itemData).length > 0 ? (
+    //아이템 데이터가 객체형식이므로 객체의 키값의 길이가 0보다클때 조건부 렌더링
+    <>
+      <div className="item-detail">
+        <div className="item-img-ul">
+          <div className="item-img-box">
+            <img src={thumbnail_images} alt="상품이미지" />
+          </div>
+          <div className="item-ul">
+            <li>
+              <i className="fa-solid fa-coins" />
+              뷰티포인트 {price / 100}P 적립
+            </li>
+            <li>
+              <i className="fa-regular fa-compass" />
+              찻잎 {price / 100}P 적립
+            </li>
+            <li>
+              <i className="fa-solid fa-truck" />
+              3만원 이상 무료배송
+            </li>
+            <li>
+              <i className="fa-solid fa-gift" />
+              (유료)포장가능
+            </li>
+          </div>
+        </div>
+        <div className="item-explain-purchase">
+          <div className="item-explain">
+            <div className="item-categories">
+              <ul className="item-categories-link">
+                <li>{first_category}</li>
+                <li>
+                  <i className="fa-solid fa-angle-right" />
+                </li>
+                <li>{second_category}</li>
+              </ul>
+            </div>
+            <div className="item-name">
+              <h3>{title}</h3>
+            </div>
+            <div className="item-name-explain">{description}</div>
+            {discount == null ? (
```

Comment:
타입 변환이 일어나지 않게 가급적이면 엄격 비교(`===`)해주시는 게 좋습니다.
그리고 삼항연산자 자체가 truthy/falsy를 검사하기 때문에 단순히 다음처럼만 작성해주셔도 됩니다.
```suggestion
            {discount ? (
```